ppr-ui/src/components/common/PartyForm.vue

```diff
@@ -15,14 +15,14 @@
             id="person-option"
             class="person-radio"
             label="Individual Person"
-            active-class="selected-radio"
+            false="selected-radio"
```

Comment:
Interesting prop `false`

Reply:
Your right, this was done by the linter automation however, I'll look into this!

Reply:
Thanks for bringing this up. 
As is, this is what the tools are telling me... although i don't quite understand it. 
Will leave as is, for now, when i get to these components will verify it's applying said class as expected or fix if not. 
Very very odd rename of a prop though i agree!

![Screenshot 2023-10-24 at 10 47 35 AM](https://github.com/bcgov/ppr/assets/53542131/ade61858-dfd4-4186-8ef5-72749e694f6c)

Reply:
Yea, ok, as long as it looks as expected.